localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -429,22 +432,31 @@ void EKFLocalizer::updateSimple1DFilters(
   z_filter_.update(z, z_dev, pose.header.stamp);
   roll_filter_.update(rpy.x, roll_dev, pose.header.stamp);
   pitch_filter_.update(rpy.y, pitch_dev, pose.header.stamp);
+  if (twist_queue_.size() > 0) {
```

Comment:
Would you consider a way not to use `twist_queue_` within this function in order to make the class as simple as possible? One possible way would be to add a `twist` argument in this function.

Reply:
Thanks for your suggestion. I agree with the opinion that `twist_queue_` here makes the code complex. 

But in the original EKF package, pose update and twist update are done separately, and updateSimple1DFilters run inside the pose update.  I am not sure what is the best way to deal with this.

Reply:
Thank you for the response. Your point is also reasonable. Indeed, the current EKF architecture does not align with the proposed algorithm.

Then how about making a new independent node for this under `localization` ?

Reply:
@kminoda 
Thanks for the detailed review. For now, I modified the algorithm to only rely on pose updates. After comprehensive consideration/examination of the source of the poses inputs, I believe poses from ndt_scan_matcher in its current form is informative enough to diagnose the bias in yaw angle (we will try doing experiments to examine the effectiveness of the algorithm first.)

Reply:
Thank you for all your considerations and quick response :pray: